lldb/source/Plugins/Process/Dpu/Dpu.cpp

```diff
@@ -654,6 +686,38 @@ bool Dpu::WriteMRAM(uint32_t offset, const void *buf, size_t size) {
   return ret == DPU_OK;
 }
 
+bool Dpu::WriteMRAM(uint32_t offset, const void *buf, size_t size) {
```

Comment:
Shouldn't we insert a doxygen-like function header comment, that explains shortly the processing difference between `Write[MW]RAM` and `Write[MW]RAMTreatingAlignment`?

Something like:

```
/**
 * @brief Write a buffer of bytes from SRAM to MRAM.
 * This function intends to use optimized write operations
 * for the part of the buffer that allows complete word-aligned read and
 * write operations
 * @param offset The offset of the first byte to be written to in MRAM
 * @buf The bytes buffer to be written from in SRAM
 * @size The number of bytes to be written
 * @return Whether the operation was succesful.
```

```
/**
 * @brief Write a buffer of bytes from SRAM to MRAM.
 * This function operates properly on buffers either word-aligned or not.
 * However, it does not attempt to fully optimize the write operations for
 * large buffers that are not properly word-aligned.
 * @param offset The offset of the first byte to be written to in MRAM
 * @buf The bytes buffer to be written from in SRAM
 * @size The number of bytes to be written
 * @return Whether the operation was succesful.
```